modules/bim/spec/seeders/root_seeder_bim_edition_spec.rb

```diff
@@ -174,12 +174,13 @@
     end
   end
 
-  describe 'demo data with a non-English language set with OPENPROJECT_DEFAULT__LANGUAGE', :settings_reset do
+  describe 'demo data with a non-English language set with OPENPROJECT_DEFAULT__LANGUAGE',
+           :settings_reset,
+           with_env: { 'OPENPROJECT_DEFAULT__LANGUAGE' => 'de' } do
```

Comment:
This does not work.

This `with_env` helper work as an around filter, but the seeding is done in a `before_all` block and the around filter is not applied yet.

I'll provide a fix for it.